modules/nested-tabs/module.php

```diff
@@ -40,7 +40,7 @@ public function __construct() {
 	 */
 	public function register_styles() {
 		$direction_suffix = is_rtl() ? '-rtl' : '';
-		$has_custom_breakpoints = Plugin::$instance->breakpoints->has_custom_breakpoints();
+		$has_custom_breakpoints = Plugin::$instance->experiments->is_feature_active( 'additional_custom_breakpoints' ) && Plugin::$instance->breakpoints->has_custom_breakpoints();
```

Comment:
Just making sure, I believe we only should load the custom widget styles, when the additional breakpoint are active?

In the original implementation, the custom files where always loaded.

Reply:
Actually no.

Elementor also loads the custom styles when the experiment is inactive, but the original desktop/mobile/mobile have different size.